packages/babel-plugin/src/utils/resolve-binding.ts

```diff
@@ -10,6 +10,7 @@ import resolve from 'resolve';
 import { DEFAULT_CODE_EXTENSIONS } from '../constants';
 import type { Metadata } from '../types';
 
+import { traverseMemberExpression } from './evaluate-expression';
```

Comment:
hmm this is still a cyclic import though, because `evaluate-expression` imports `resolve-binding` and vice versa.